src/Api/OpenApi/OpenApiDecorator.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @file
+ * OpenApi Service Decorator
+ *
+ * @see https://api-platform.com/docs/core/swagger/#overriding-the-openapi-specification
+ */
+
+namespace App\Api\OpenApi;
+
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * Class OpenApiDecorator.
+ */
+final class OpenApiDecorator implements NormalizerInterface
+{
+    private $decorated;
+
+    /**
+     * OpenApiDecorator constructor.
+     *
+     * @param NormalizerInterface $decorated
+     */
+    public function __construct(NormalizerInterface $decorated)
+    {
+        $this->decorated = $decorated;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function normalize($object, $format = null, array $context = [])
+    {
+        $docs = $this->decorated->normalize($object, $format, $context);
+
+        // We don't expose a item endpoint but api-platform requires one,
+        // so we need to manually remove it from the docs.
+        unset($docs['paths']['/api/v2/covers/{id}']);
```

Comment:
The `/api/v2` is configurable and the `APP_PATH_PREFIX` should be used to unset this.